File: src/main/java/lesson2/Cru.java
package lesson2;

public enum Cru {
    CRU,
    PREMIER,
    GRAND();

    Cru() {
        System.out.println("CRU" + this.name());
    }

    public String toString() {
        return "lesson1And2.Cru{}";
    }
}

class Test {
    public static void main(String[] args) {
        System.out.println(Cru.CRU);
    }
}

File: src/main/kotlin/lesson2/classes/Cru.kt
package lesson2.classes

enum class Cru(val lvl: Int) {
    CRU(80),
    PREMIER(90),
    GRAND(100);

    override fun toString() =
        when (this) {
            CRU -> "Base vine"
            PREMIER -> "Premier vine"
            GRAND -> "Grand vine"
        }

}

fun main() {
//    println("lesson1And2.Cru: ${lesson1And2.Cru.CRU.lvl}")
//    println("PREMIER: ${lesson1And2.Cru.PREMIER.lvl}")
//    println("Grand: ${lesson1And2.Cru.GRAND.lvl}")
//    println(lesson1And2.Cru.GRAND)
//    println("Grand: ${lesson1And2.Cru.GRAND.lvl}")
//    println(lesson1And2.Cru.GRAND)

    Cru.entries.forEach { println(it) }


}